release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2078,7 +2078,7 @@ $ ./openshift-install wait-for install-complete
 
 * Due to the inclusion of old images in some image indexes, running `oc adm catalog mirror` and `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
 
-* In {product-title} on {rh-openstack}, you can assign a floating IP address to a reservation port to have a predictable SNAT address for egress traffic. Currently, when an egress IP address on an {rh-openstack} network moves from one node to another because of a node failover, for example, the neutron reservation port is removed and recreated. However, when an administrative user associates a floating IP address with a neutron reservation port, the delete operation hangs due to insufficient privileges. The administrative user must unset the floating IP address association manually to resolve the issue. (link:https://issues.redhat.com/browse/OCPBUGS-4902[*OCPBUGS-4902*])
+* Egress IP addresses ensure that services outside of the cluster network have consistent source IP addresses. In {product-title} on {rh-openstack}, you can assign a floating IP address to a reservation port to have a predictable SNAT address for egress traffic. Currently, when an egress IP address on an {rh-openstack} network moves from one node to another, the neutron reservation port is removed and recreated. The administrative user that installed the cluster must have access to the floating IP addresses that are assigned to the reservation ports. If not, the egress IP addresses cannot be deleted. (link:https://issues.redhat.com/browse/OCPBUGS-4902[*OCPBUGS-4902*])
```

Comment:
"Egress IP addresses ensure that services outside of the cluster network have consistent source IP addresses." - this isn't exactly true, by default EgressIP in OpenStack affects traffic inside the cluster only. Let's go with @andreaskaris suggestion.